code/modules/clothing/accessories/holomap.dm

```diff
@@ -221,8 +247,65 @@ var/list/holomap_cache = list()
 			animate(alpha = 255, time = 2, easing = SINE_EASING)
 			holomap_images += I
 
+	//Additional things we might want to track
+	extra_update()
+
 	activator.client.images |= holomap_images
 
+/obj/item/clothing/accessory/holomap_chip/proc/extra_update()
+	return
+
+/obj/item/clothing/accessory/holomap_chip/deathsquad/extra_update()
+	var/turf/T = get_turf(src)
+	for(var/obj/mecha/combat/marauder/maraud in mechas_list)
+		if(!istype(maraud,/obj/mecha/combat/marauder/series) && !istype(maraud,/obj/mecha/combat/marauder/mauler) && (T.z == maraud.z))//ignore custom-built and syndicate ones
+			var/i = 0
```

Comment:
i is a bad name for any variable used outside of a loop. You can take out the var use entirely using a ternary.